x-pack/plugins/apm/public/components/shared/charts/instances_latency_distribution_chart/index.tsx

```diff
@@ -46,11 +50,27 @@ export function InstancesLatencyDistributionChart({
     bubbleSeriesStyle: {
       point: { strokeWidth: 0, fill: theme.eui.euiColorVis1, radius: 4 },
     },
+    // This additional padding makes it so items with small values don't look like
+    // zeroes right up against the origin.
+    chartPaddings: { top: 0, left: 10, right: 10, bottom: 10 },
   };
 
   const maxLatency = Math.max(...items.map((item) => item.latency ?? 0));
   const latencyFormatter = getDurationFormatter(maxLatency);
 
+  const tooltip: TooltipProps = {
+    type: TooltipType.Follow,
+    snap: false,
+    customTooltip: (props: TooltipInfo) => (
+      <CustomTooltip {...props} latencyFormatter={latencyFormatter} />
+    ),
+  };
+
+  // When we only have a single point, we want to use an ordinal scale instead
+  // of linear because an ordinal scale will place the point in the middle, while
+  // linear will place it at the origin.
+  const xScaleType = items.length === 1 ? ScaleType.Ordinal : ScaleType.Linear;
```

Comment:
I'm not sure this is the right solution. I know we want to fix this case:
![image](https://user-images.githubusercontent.com/209966/113060312-c298d680-91b0-11eb-9c61-a7b92206dafb.png)


But what happens if there are two instances, both with 7.6tpm? Or just similar throughput?

In either case, I think we always want to lower bound to be 0.